src/main/docker/Dockerfile

```diff
@@ -1,43 +1,42 @@
-FROM openjdk:8u201-jre-alpine
+FROM alpine:3.10
 LABEL maintainer="steve.springett@owasp.org"
 LABEL vendor="OWASP"
 
 # Arguments that can be passed at build time
 ARG APP_DIR=/opt/owasp/dependency-track
 ARG DATA_DIR=/data
+ARG EXTLIB_DIR=/extlib
```

Comment:
extlib was the only hardcoded directory so thought it'd be good to parametrise it too.